packages/fxa-auth-server/package.json

```diff
@@ -29,7 +29,11 @@
     "paypal-processor": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts",
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts",
     "write-templates": "ts-node lib/senders/emails/render-to-disk.ts",
-    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts"
+    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts",
+    "scss": "node-sass lib/senders/emails/scss -o lib/senders/emails/css",
```

Comment:
We also wanted to verify if we can use `scss` for our templates so yes we probably can by setting up a build pipeline similar to this

Reply:
If we keep this command it should be renamed to something more specific, like `email-scss`